### DISCUSSION TITLE
Asus Z13 Acronym - Airplane Mode Always On When Boot

### DISCUSSION THREAD

**Discussion by BChip** [seerge/g-helper#808] [Q&A]
Hello everyone,

Quick question, why does my Z13 keep booting in airplane mode? No matter what I do, it is always in airplane mode when i first boot. I can disable airplane mode and it connects to wifi just fine. I just don't understand why it does it everytime. I did go into Windows Event Viewer and here is some info on why it might be. Any suggestions on how to fix this?

```
The description for Event ID 6062 from source Netwtw12 cannot be found. Either the component that raises this event is not installed on your local computer or the installation is corrupted. You can install or repair the component on the local computer.

If the event originated on another computer, the display information had to be saved with the event.

The following information was included with the event: 

\Device\NDMP1
Intel(R) Wi-Fi 6E AX211 160MHz

The message resource is present but the message was not found in the message table
```

I am using the latest Intel Wifi drivers.

I have uninstalled the drivers and re-installed the drivers.

**Comment by seerge** [comment]
@BChip does it also happen if g-helper is not set to run on startup ? If so - looks like your windows is acting. May be it's some tricky setting ? Otherwise you can probably set some short powershell script to run on startup disabling airplane mode? :)

**Comment by TemperoraryUsernameWillChangeLater** [comment]
I have the exact same issue. Have you found a fix?
